naga/src/front/wgsl/error.rs

```diff
@@ -1370,6 +1383,27 @@ impl<'a> Error<'a> {
                 ],
                 notes: vec![],
             },
+            Error::MissingMeshShaderInfo { mesh_attribute_span} => ParseError {
+                message: "mesh shader entry point is missing @vertex_output or @primitive_output".into(),
```

Comment:
suggestion: It seems like both of these are necessary, so let's make that clear in this diagnostic.

```suggestion
                message: "mesh shader entry point is missing both @vertex_output and @primitive_output".into(),
```